### PR TITLE
Meshnet plan

### PR DESCRIPTION
**Pull request by benhylau**

### CODE REVIEW

File: meshnet/README.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+- Access points in key areas for client devices (e.g. phones and laptops) to connect
+- Allow Camp participants to extend or run locally-hosted services on the network
+- Carry traffic to Internet gateway for client devices to access the Internet with limited bandwidth and reliability
+- Document setup and learnings for future events or other groups
+
+## Hardware
+
+The diagram above is the area map, with preliminary positions of _mesh nodes_ that together form the local mesh network.
+Each node has a 100 m radius circle around it, representing the region where client devices are expected to form a good connection with the node's access point.
+There is a larger 150 m radiu circle around where devices may be able to connect. The green dotted lines represent point-to-point links between nodes that together form the local mesh network.
```
Comment (gobengo) [inline meshnet/README.md]: s/radiu circle/radius circle/

File: meshnet/README.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+Each node is an aggregate of multiple pieces of equipment, which may include:
+
+- Radio for directional link
+- Radio for access point
+- Network switch, router, and application server
+- Power equipment, mounts, and cables
+
+Some nodes will have interfaces that allow other devices to be plugged in to offer locally-hosted services on the network.
```
Comment (gobengo) [inline meshnet/README.md:36]: Will/should I be able to host a service on the network from my laptop that is connected to the network?

Or will I need to bring a microcomputer (e.g. rpi) and plug into one of these official planned nodes?

File: meshnet/README.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+- Radio for access point
+- Network switch, router, and application server
+- Power equipment, mounts, and cables
+
+Some nodes will have interfaces that allow other devices to be plugged in to offer locally-hosted services on the network. 
+One or more nodes will also serve as the Internet gateway for this local network.
+
+### Radio for directional link
+
+Each green dotted line will be formed a pair of [MikroTik SXTsq 5 ac](https://mikrotik.com/product/sxtsq_5_ac), [used in NYC Mesh](https://docs.nycmesh.net/hardware/sxtsqg5acd/), supporting link speeds at 800+ Mbps. Where possible, we can also link two nodes with a long ethernet cable.
```
Comment (gobengo) [inline meshnet/README.md]: s/formed/formed by/

(maybe. I think that's what you mean)

File: meshnet/README.md (hunk 2 above)
Comment (benhylau) [reply meshnet/README.md:36]: Yes but IP allocation hasn't been well thought out. Adding to this https://github.com/dweb-camp-2019/organizing/issues/17

**Pull request merged**